src/components/layout/header/utils/index.js

```diff
@@ -0,0 +1,131 @@
+import { api_base } from '@/external/bot-skeleton';
+
+export const LOW_RISK_COUNTRIES = () => ['za', 'ec', 'bw'];
+export const EU_COUNTRIES = () => [
+    'it',
+    'de',
+    'fr',
+    'lu',
+    'gr',
+    'mf',
+    'es',
+    'sk',
+    'lt',
+    'nl',
+    'at',
+    'bg',
+    'si',
+    'cy',
+    'be',
+    'ro',
+    'hr',
+    'pt',
+    'pl',
+    'lv',
+    'ee',
+    'cz',
+    'fi',
+    'hu',
+    'dk',
+    'se',
+    'ie',
+    'im',
+    'gb',
+    'mt',
+];
+
+export const isEu = country_code => EU_COUNTRIES().includes(country_code);
+
+export const isEuByAccount = (account = {}) => {
+    const { loginInfo = {} } = account;
+    return EU_COUNTRIES().includes(loginInfo.country);
+};
+
+export const isEuLandingCompany = landing_company => /^(maltainvest|malta|iom)$/.test(landing_company);
+
+export const hasEuAccount = token_list =>
+    token_list.some(token_obj => isEuLandingCompany(token_obj.loginInfo.landing_company_name));
+
+const isLowRisk = (financial, gaming, upgrade) => {
+    const { shortcode: f_sc = '' } = financial || {};
+    const { shortcode: g_sc = '' } = gaming || {};
+    return (f_sc === 'maltainvest' && g_sc === 'svg') || (upgrade?.includes('svg') && upgrade?.includes('maltainvest'));
+};
+
+const isHighRisk = (financial_company, gaming_company, risk_classification) => {
+    const restricted_countries =
+        financial_company?.shortcode === 'svg' ||
+        (gaming_company?.shortcode === 'svg' && financial_company?.shortcode !== 'maltainvest');
+
+    const high_risk_landing_company = financial_company?.shortcode === 'svg' && gaming_company?.shortcode === 'svg';
+    return risk_classification === 'high' || high_risk_landing_company || restricted_countries;
+};
+
+const isMultiplier = landing_company_list => {
+    const multiplier_account = landing_company_list?.financial_company?.legal_allowed_contract_categories;
+    const is_multiplier = multiplier_account?.includes('multiplier');
+    return {
+        is_multiplier: multiplier_account?.length === 1 && is_multiplier,
+        country_code: landing_company_list.id,
+    };
+};
+
+export const checkSwitcherType = async (client_accounts = {}, activeLoginid, isVirtual) => {
+    const virtual_accounts = [];
+    const non_eu_accounts = [];
+    const eu_accounts = [];
+    const real_accounts = [...eu_accounts, ...non_eu_accounts];
+    if (!activeLoginid) return null;
+    const account_info = { ...api_base.account_info };
+
+    if (!account_info) return null;
+    const landing_company = await api_base.getLandingCompany();
```

Comment:
Let's not make the call again. We already have the data in store. Let's pass it to this function.